### DISCUSSION TITLE
combining "domain adaptation" technique and chemprop

### DISCUSSION THREAD

**Discussion by SoodabehGhaffari** [chemprop/chemprop#917] [General]
Hello,
Previously, I asked a question about transfer learning and comparing it with modeling from scratch. I tried the pretrained models in chemprop to build a model for a related task. However, I realized the transfer learning model is not alway better than the models from scratch. Recently, I came across with a new method called "domain adaptation". I was wondering if chemprop has the feature to do "domain adaptation" and if there is any related paper in this regard.

I really appreciate your help.

Best Regards
Soo

**Comment by KnathanM** [comment]
To my knowledge, there aren't any chemprop papers about domain adaptation, but an internet search of "domain adaptation chemistry" comes up with a couple of general results. 

Whether it will improve model performance to initialize the weights from another model trained on a different task is usually hard to predict. Best to try it and see what the empirical results are.
